### DISCUSSION TITLE
Google Play Closed Testing

### DISCUSSION THREAD

**Discussion by danemadsen** [Mobile-Artificial-Intelligence/maid#499] [Announcements]
Testing is now complete. I am now going through the process of finalizing a public release

Maid is also now available on FDroid.

**Comment by XMHNHMX** [comment]
How to get on the testing list?

**Comment by danemadsen** [comment]
If you email me from your google a accounts email I can add you to the closed testing list. I can't do open testing until I have 20 people in closed testing unfortunately.

**Comment by narakai** [comment]
I can help with the close testing, have sent an email to you.

**Reply by danemadsen** [reply]
Yes sorry I'll add you soon just abit busy right now

**Comment by danemadsen** [comment]
@narakai @XMHNHMX  Ok you have both been added to the closed testing list.

**Comment by madsaks** [comment]
I've emailed.

**Reply by danemadsen** [reply]
Yep added

**Reply by liljohnak** [reply]
@danemadsen I can test; email sent.

**Reply by danemadsen** [reply]
> @danemadsen I can test; email sent.

Yep I've added you to the list. You can opt in here

https://play.google.com/apps/testing/com.danemadsen.maid

**Comment by Jafner** [comment]
Email sent.  Cool project!

**Reply by danemadsen** [reply]
Yep added:

https://play.google.com/apps/testing/com.danemadsen.maid

**Comment by ainfosecchimp** [comment]
Sounds like a top project, can I help? Plus beta test please 🙂

**Reply by danemadsen** [reply]
Sure you can help. Just send me an email from your google play connected email (or gmail) and I can add you.

**Comment by rodion-m** [comment]
Wow, why did Google deny production access for your app?

**Comment by danemadsen** [comment]
It got approved on the second try. Just sorting out the release now. It should release on google play shortly after I release 1.2.9 which will be some time this week.

With regards,
Dane Madsen
________________________________

**Comment by mindrunner** [comment]
I sent you an email :)
